lms/templates/commerce/receipt.underscore

```diff
@@ -0,0 +1,94 @@
+              </tr>
+            </thead>
+
+            <tbody>
+              <% for ( var i = 0; i < receipt.items.length; i++ ) { %>
+                <% if ( receipt.isRefunded ) { %>
+                  <td><del><%- receipt.orderNum %></del></td>
+                  <td><del><%- receipt.items[i].lineDescription %></del></td>
+                  <td><del><%- receipt.purchasedDatetime %></del></td>
+                  <td><del><%- receipt.items[i].cost %> (<%- receipt.currency.toUpperCase() %>)</del></td>
+                <% } else { %>
+                  <tr>
+                    <td><%- receipt.orderNum %></td>
+                    <td><%- receipt.items[i].lineDescription %></td>
+                    <td><%- receipt.purchasedDatetime %></td>
+                    <td><%- receipt.items[i].cost %> (<%- receipt.currency.toUpperCase() %>)</td>
+                  </tr>
+                <% } %>
+              <% } %>
+            </tbody>
+
+            <tfoot>
+              <tr>
+                <th scope="row" class="total-label" colspan="1"><%- gettext( "Total" ) %></th>
+                <td class="total-value" colspan="3">
+                  <span class="value-amount"><%- receipt.totalCost %></span>
+                   <span class="value-currency">(<%- receipt.currency.toUpperCase() %>)</span>
+                </td>
+              </tr>
+            </tfoot>
+          </table>
+
+          <% if ( receipt.isRefunded ) { %>
+          <div class="msg msg-refunds">
+            <h4 class="title sr"><%- gettext( "Please Note" ) %>: </h4>
+            <div class="copy">
+              <p><%- gettext( "Crossed out items have been refunded." ) %></p>
+            </div>
+          </div>
+          <% } %>
+        </div>
+
+        <div class="copy">
+          <p><%- gettext( "Billed to" ) %>:
+              <span class="name-first"><%- receipt.billedTo.firstName %></span>
+              <span class="name-last"><%- receipt.billedTo.lastName %></span>
+              (<span class="address-city"><%- receipt.billedTo.city %></span>,
+              <span class="address-state"><%- receipt.billedTo.state %></span>
+              <span class="address-postalcode"><%- receipt.billedTo.postalCode %></span>
+              <span class="address-country"><%- receipt.billedTo.country.toUpperCase() %></span>)
+          </p>
+        </div>
+      </li>
+    </ul>
+    <% } else { %>
+        <p class="no-content"><%- gettext( "No receipt available" ) %></p>
+    <% } %>
+
+    <nav class="nav-wizard is-ready">
+
+      <a id="verify_later_button" class="next action-secondary verify-later nav-link" href="/dashboard" data-tooltip="<%- _.sprintf( gettext( "If you don't verify your identity now, you can still explore your course from your dashboard. You will receive periodic reminders from %(platformName)s to verify your identity." ), { platformName: platformName } ) %>">
+        <%- gettext( "Want to confirm your identity later?" ) %>
+      </a>
```

Comment:
Does this need to be controlled by an if statement?  It shouldn't be displayed if (a) the user has submitted a verification that hasn't expired, been denied, or errored, or (b) the course mode does not require verification (currently only for no-id prof ed)